src/reframe.js

```diff
@@ -21,8 +21,8 @@ export default function reframe(target, cName) {
   }
 }
 
-if (window.$) {
```

Comment:
@mohandere I'm declining this pr. An updated fix for your note will be added soon. In the meantime - just use `reframe(el);` ~Thanks!